staging/src/k8s.io/cloud-provider/controllers/service/metrics.go

```diff
@@ -32,12 +33,26 @@ var register sync.Once
 // registerMetrics registers service-controller metrics.
 func registerMetrics() {
 	register.Do(func() {
+		legacyregistry.MustRegister(loadBalancerSyncCount)
 		legacyregistry.MustRegister(nodeSyncLatency)
+		legacyregistry.MustRegister(nodeSyncErrorCount)
 		legacyregistry.MustRegister(updateLoadBalancerHostLatency)
 	})
 }
 
 var (
+	loadBalancerSyncCount = metrics.NewCounter(&metrics.CounterOpts{
+		Name:           "load_balancer_sync_total",
+		Subsystem:      subSystemName,
+		Help:           "A metric counting the amount of times any load balancer has been configured, as an effect of service/node changes on the cluster",
+		StabilityLevel: metrics.ALPHA,
+	})
+	nodeSyncErrorCount = metrics.NewCounter(&metrics.CounterOpts{
+		Name:           "node_sync_error_total",
```

Comment:
Below metric writes "nodesync" together. I would recommend to keep it consistent either use `nodesync_error_total` or also update `nodesync_latency_seconds` to `node_sync_latency_seconds`. Those are Alpha metrics so rename is not a problem as long as it mentioned in changelog.

Reply:
OK, I went with `nodesync_error_total ` as to not break any user/consumer depending on or using `nodesync_latency_seconds`